src/less/signals/base/signals.less

```diff
@@ -0,0 +1,37 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+.signal {
```

Comment:
Let's use a pattern that we've used in other modules (e..g field), where a span.signal will be display: inline-block, and a div.signal will be display: block.

Reply:
Actually, scrap that idea. Let's keep this simple and just have `.signal { display: inline-block }` (so will always be inline block regardless of span or div)

This seems like good default behaviour in case anything needs to go alongside a signal:

![Screen Shot 2021-03-30 at 3 15 51 PM](https://user-images.githubusercontent.com/38065/113064179-5e075880-916b-11eb-88fc-3a4a45f62593.png)